src/components/Editor/DraggableTabs/DroppableTabList.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import { Droppable } from "@hello-pangea/dnd"
+import { TabList } from "react-tabs"
+
+import './DraggableTabs.scss'
+
+const DroppableTabList = ({children, index, ...otherProps}) => {
+  return (
+    <TabList {...otherProps }>
+      <Droppable direction='horizontal' droppableId={`${index}`}>
+        {(provided) => (
+          <div className = 'droppable-tab-list' {...provided.droppableProps} ref={provided.innerRef}>
```

Comment:
as with my comment on the DraggableTab component, you might want to use destructuring on the params?